File: MD-Arrays/1-Missing_Number_In_Array/soln.cpp
#include <iostream>
using namespace std;

int main() {
    int t;
    cin >> t;
    while (t--){
        int n;
        cin >> n;
        int arr[n];
        int sum_n = n*(n+1)/2;
        int arr_sum = 0;
        for (int i=1; i<n; i++){
            cin >> arr[i];
            arr_sum = arr_sum + arr[i];
        }
        cout << (sum_n - arr_sum) <<endl;
    }
	//code
	return 0;
}

File: MD-Arrays/1-Missing_Number_In_Array/soln.py

def FindMissNum(arr,n):
    sum_n = n*(n+1)/2
    arr_sum = sum(arr)
    return int(sum_n-arr_sum)

t = int(input())

while t>0:
    n=int(input())
    arr=list(map(int,input().split()))
    print(FindMissNum(arr,n))
    t=t-1
    
